### DISCUSSION TITLE
Is it possible to take over the "Cadwyn" name?

### DISCUSSION THREAD

**Discussion by zmievsa** [community/community#113465] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I see that github.com/Cadwyn account has only been used in 2020-2021 a few times but no usage since then.

Is it possible to somehow take over that name? I know the policy on name squatting but see that this account has clearly been created for some specific project and left unused since then. I would like to use it for my organization for a group of projects around github.com/zmievsa/cadwyn

**Comment by ghostinhershell** [comment]
Hi @zmievsa,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!

**Reply by zmievsa** [reply]
Understood, thank you! Is it maybe possible to ask Github to contact the owner of that account? He left no contact information and I am sure he would be willing to discuss the name transfer.
